src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+  const total = days * dayPrice;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days < 3) {
+    return total;
```

Comment:
Consider using a constant for the minimum days required to receive a discount, to avoid hardcoding and make the code more maintainable. Refer to the checklist item 2.